src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSource.java

```diff
@@ -831,10 +831,16 @@ public ListBoxModel doFillProjectPathItems(@AncestorInPath SCMSourceOwner contex
             @QueryParameter String serverName,
             @QueryParameter String projectOwner) {
             ListBoxModel result = new ListBoxModel();
```

Comment:
Move this down 😅 below the size if statement to save creating this object at later having to discard it.